pkg/resources/testdata/TestAcc_ExternalVolume/azure/with-storage-aws-role-arn/test.tf

```diff
@@ -0,0 +1,15 @@
+resource "snowflake_external_volume" "complete" {
+  name         = var.name
+  comment      = var.comment
+  allow_writes = var.allow_writes
+  dynamic "storage_location" {
+    for_each = var.storage_location
+    content {
+      storage_location_name = storage_location.value["storage_location_name"]
+      storage_provider      = storage_location.value["storage_provider"]
+      storage_base_url      = storage_location.value["storage_base_url"]
+      azure_tenant_id       = storage_location.value["azure_tenant_id"]
+      storage_aws_role_arn  = storage_location.value["storage_aws_role_arn"]
```

Comment:
According to the documentation, we can't have azure provider with AWS ROLE ARN.

Reply:
Yep, this is to test that this config results in a failure - https://github.com/Snowflake-Labs/terraform-provider-snowflake/blob/afbd4a00ec193042456b0f2a2c1bf99b3b494f4a/pkg/resources/external_volume_acceptance_test.go#L1911

Reply:
Sorry, I didn't notice. In this case, I would rename this directory to indicate that it contains invalid configurations. The same for other such directories.

Reply:
Ended up deleting this in https://github.com/Snowflake-Labs/terraform-provider-snowflake/commit/68c104d594d26823233d5aeef3767c359004d37f as I consolidated many test directories